### DISCUSSION TITLE
`App\Http\Controllers\Spatie\WebhookClient\Http\Controllers\WebhookController` was not found:

### DISCUSSION THREAD

**Discussion by ash2050** [spatie/laravel-webhook-client#118] [Q&A]
Hi

I am trying to use webhook client but seem to be failing utterly... 

I've followed the github docs but 3rd party api cannot get successful 200 when using Route::webhook, but is successful when I use Route::Post.  Its chucking out  

  Invalid route action: [App\Http\Controllers\Spatie\WebhookClient\Http\Controllers\WebhookController].

  • `App\Http\Controllers\Spatie\WebhookClient\Http\Controllers\WebhookController` was not found: Controller class `App\Http\Controllers\Spatie\WebhookClient\Http\Controllers\WebhookController` for one of your routes was not found. Are you sure this controller exists and is imported correctly?

I cant log json out from the job i've created and cannot tell if its just not accepting the post request or i ve completey made a hash of implemtation.

Cheers.

**Comment by froday** [comment]
Did you resolve this? I am having the same issue.

**Reply by kl4ver** [reply]
I have the same issue, so I downgraded to the 2.7.5 release.

**Reply by ash2050** [reply]
So, did downgrading work?

**Reply by froday** [reply]
Yeah it worked for me

**Reply by ash2050** [reply]
Wonderful.  

Still would like to understand why this happened?.!

Hopefully someone could clue us in at some point.

**Comment by eric-hansen** [comment]
In case someone else is having this issue, it seems to have to do if you have `./app/Providers/RouteServiceProvider.php`'s property `$namespace` populated.

Mine was defined like this:

```
protected $namespace = 'App\Http\Controllers';
```

Once I commented it out I was not getting this error anymore.

**Reply by eric-hansen** [reply]
Another thing to check is where you may be defining the webhooks routes.  I also ended up having to remove this from my `./routes/api.php` file:

```
Route::prefix('webhooks')->group(function()
{
    Route::webhooks('/theuri');
});
```
